src/cdk/testing/protractor/protractor-harness-environment.ts

```diff
@@ -37,6 +37,10 @@ export class ProtractorHarnessEnvironment extends HarnessEnvironment<ElementFind
     return new ProtractorHarnessEnvironment(protractorElement(by.css('body')), options);
   }
 
+  static getNativeElement(el: ProtractorElement): ElementFinder {
```

Comment:
This should take in a `TestElement` and just assert that its a `ProtractorElement`:
```ts
if (el instanceof ProtractorElement) {
  return el.element;
}
throw Error('This TestElement was not created by the ProtractorHarnessEnvironment');
```
That way users can avoid annoying casting.

Reply:
I'll do this for `testbed-harness-environment.ts`, too